.python-version

```diff
@@ -0,0 +1 @@
+2.7.16
```

Comment:
I'm using pyenv and figure that while I'm in here, may as well add a version we know works to the repo. Happy to back out this change though, since it's unrelated to this specific PR.